test/e2e/cluster_size_autoscaling.go

```diff
@@ -262,20 +265,20 @@ func isAutoscalerEnabled() (bool, error) {
 	strBody := string(body)
 	glog.Infof("Cluster config %s", strBody)
 
-	if strings.Contains(strBody, "minNodeCount") {
+	if strings.Contains(strBody, "\"minNodeCount\": "+strconv.Itoa(expectedMinNodeCountInTargetPool)) {
 		return true, nil
 	}
 	return false, nil
 }
 
-func enableAutoscaler() error {
+func enableAutoscaler(nodePool string, minCount, maxCount int) error {
 	updateRequest := "{" +
 		" \"update\": {" +
-		"  \"desiredNodePoolId\": \"default-pool\"," +
+		"  \"desiredNodePoolId\": \"" + nodePool + "\"," +
```

Comment:
I'm going to merge this since it's apparently important, but please when you get a chance switch this code to use backticks, it'll be much more readable.